ci-operator/jobs/openshift/cred-minter/openshift-cred-minter-master-postsubmits.yaml

```diff
@@ -0,0 +1,34 @@
+postsubmits:
+  openshift/cred-minter:
```

Comment:
Naming wise this should probably be credential-manager or cloud-credential-operator

Reply:
I emailed about the naming on Dec 6th as a followup to the arch call where we discussed the minter, but got no objections. I now have repos created and in flight for both this and an operator, dist-git and ART requests filed. Should I pull the plug on all of it to rename bearing in mind we're trying to make dev freeze?